src/pages/das-api/methods/get-nft-editions.md

```
@@ -0,0 +1,35 @@
+
+The response includes:
+
+- `editions` - Array of edition objects containing:
+  - `edition_address` - The address of the [edition account](/token-metadata#printing-editions)
+  - `edition_number` - The edition number (1, 2, 3, etc.)
+  - `mint_address` - The mint address of the edition
+- `master_edition_address` - Address of the master edition account
+- `supply` - Current number of editions minted
+- `max_supply` - Maximum number of editions that can be minted (null for unlimited)
+
+
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)